google/cloud/storage/bucket.h

```diff
@@ -56,10 +55,28 @@ class Bucket {
     return GetMetadataImpl(request);
   }
 
+  /**
+   * Create an object given its name and media (contents).
+   *
+   * @par Example
+   * @snippet storage_bucket_samples.cc insert object
+   *
+   * TODO(#682) - prototype modifiers for the request.
+   */
+  template <typename... Modifiers>
+  ObjectMetadata InsertObject(std::string const& object_name,
+                              std::string contents, Modifiers&&... modifier) {
```

Comment:
Nit: why "modifier" instead of "modifiers"? This could represent several args, right?

Reply:
Fixed.